schema.sql

```diff
@@ -1,5 +1,18 @@
-/* Database schema to keep the structure of entire database. */
+-- Table: public.animals
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- DROP TABLE IF EXISTS public.animals;
+
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer,
```

Comment:
- [ OPTIONAL ] Please note that you make the id column not null and autoincremented by changing you existing code to:
```
id int NOT NULL IDENTITY(1,1)
```

Reply:
Done